README.md

```diff
@@ -0,0 +1,48 @@
+# Project Name: Portfolio Project
+
+## Description:
+
+This project is based on html and css technology to build and design a portfolio website. The project was implemented with linters which make sure that some specific features are implemented successfully on the project for best user experience.
+
+The purpose of the project is to create a website that has the best user experience. The html document has an header that displays portfolio details while the css file which is the style document for the project has a styling color that beautify the html document's structure. The linters in this project includes:
+"lighthouse" which help to improve the quality of webpage,
+"webhint" is a customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors and
+"Stylelint" which helps you avoid errors and enforce conventions in your styles.
+The goal here is to master all of the tools and best practices because they will be used more often throughout Microverse projects and most likely in future job as well.
+
+## Built With
+
+Major languages: html, css
+
+## Getting Started
+
+- The feature branch was first created
+- then linter.yml file was added to the .github/workflow folder in feature branch
+  -the added file was staged using the git command
+- the featured branch was pushed to the remote repository
+- pull request was opened on the remote repository which makes the linters file failed some test
+- then the linter was set up properly by installing its dependencies
+- after the linter was properly set up, the html file was created
+- the whole project was properly set up in the html file
+- after the html file is being set up, the style file was added
+- the necessary styling was added to the style file.
+
+## Author
+
+👨 Abimbola Adedeji
+GitHub: @fayob
+Twitter: @Fabimworld2536
+LinkedIn: LinkedIn.com/in/abimbola-ade
```

Comment:
Also, it would be a good idea to add your social handle links here, so visitors can find you on other platforms easily. You know, just these little things that make your profile better.

Reply:
They are all my social media links